packages/wallet-management/src/types/index.ts

```diff
@@ -27,17 +30,17 @@ export interface AccountData {
   chainId: number;
   address: string;
   signer: Signer;
-  provider: Web3Provider;
+  provider: ethers.providers.Web3Provider;
+  isSafeWallet?: boolean;
+  sdk?: SafeAppsSDK;
```

Comment:
Not sure how to generalize this field, but probably we should try to find a way.

Reply:
we aren't using the sdk from this place anyways, it's being accessed from the instance present inside the provider. I'll take a look at this and if required remove it